<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>soldout</class>
 <widget class="QDialog" name="soldout">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sold out Box</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget {
    background-color: rgb(130, 163, 255);  
}

QDateEdit {
     border: 2px solid #ccc; 
    border-radius: 3px; 
    padding: 5px; 
    font-size: 14px; 
    color: rgb(165, 0, 0);
    background-color: rgb(129, 200, 255);
font-weight : bold;
}

QPushButton:hover {
    background-color: #c0392b; 
}

QLineEdit {
    border: 2px solid #ccc; 
    border-radius: 3px; 
    padding: 5px; 
    font-size: 14px; 
    color: rgb(165, 0, 0);
    background-color: rgb(129, 200, 255);
font-weight : bold;
}

QLabel {
  color:rgb(0, 0, 0);
 font-size: 18px;
	
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Box Number :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Selling date :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Selling amount :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="boxNumberIn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="sellAmountIn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QDateEdit" name="sellDateIn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2024</year>
     <month>10</month>
     <day>10</day>
    </date>
   </property>
  </widget>
  <widget class="QPushButton" name="soldOutBt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border:none;
color:white;
background-color:rgb(0, 108, 0);
border-radius: 7px;

}

QPushButton:hover {
    background-color: rgb(0, 34, 0); 
}</string>
   </property>
   <property name="text">
    <string>Sold Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResetBt">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border:none;
color:white;
background-color:red;
border-radius: 7px;
}

QPushButton:hover {
    background-color: rgb(85, 0, 0); 
}</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="showResult">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lima Bosonto Unicode</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    border: 4px solid #ccc; 
    border-radius: 10px; 
    padding: 5px; 
    font-size: 14px; 
    color: rgb(165, 0, 0);
    background-color: rgb(129, 200, 255);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/main1/Picture3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>boxNumberIn</zorder>
  <zorder>sellAmountIn</zorder>
  <zorder>sellDateIn</zorder>
  <zorder>soldOutBt</zorder>
  <zorder>ResetBt</zorder>
  <zorder>showResult</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
